docs/components/slideout.md

```diff
@@ -68,7 +68,11 @@ Controls the close button alignment, can be `start` (default) or `end`.
 
 ### offsetTop
 
-Allows a host app to define the offset from the top of the page, expects a `number` and that will be converted to `px`, defaults to `0px`.
+* **Type**: `number | string`
+* **Required**: no
+* **Default**: `0`
```

Comment:
(Unfortunately) we don't document any other of the props in this format for this component; can you either 1) back out this doc format, or 2) update the format for all documented props?

Reply:
I added the same style of documentation to the remaining props in KSlideout in https://github.com/Kong/kongponents/pull/1769/commits/2b213e7b8e160f8674ba6e22292b5f66e522314f.